infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/CorfuInterClusterReplicationServer.java

```diff
@@ -271,6 +272,9 @@ private void startServer() {
         }
 
         log.info("main: Server exiting due to shutdown");
+        // Flush the async appender before exiting to prevent the loss of logs
```

Comment:
can we move this to a flushAsyncAppender() function?

Reply:
sure. done